fluxfw-gcp/img/loginbuild.yaml

```diff
@@ -37,9 +37,12 @@ steps:
   - source_image=${_SOURCE_IMAGE}
   - -var
   - source_image_project_id=${_SOURCE_IMAGE_PROJECT_ID}
+  - -var
+  - subnetwork=${_SUBNETWORK}
   - login.pkr.hcl
 substitutions:
   _SOURCE_IMAGE: rocky-linux-8-optimized-gcp-v20220719
   _SOURCE_IMAGE_PROJECT_ID: rocky-linux-cloud 
+  _SUBNETWORK: default
```

Comment:
So, the identical variable is used to substitute in the YAML as well as in the build? I'm confused by this.

Reply:
It's an unfortunate aspect of Cloud Build configs.